primitives/core/src/assertion.rs

```diff
@@ -52,6 +52,7 @@ pub enum Assertion {
 	A10(ParameterString),                                   // (minimum_amount)
 	A11(ParameterString),                                   // (minimum_amount)
 	A13(u32),                                               // (Karma_amount) - TODO: unsupported
+	TwitterFollower(ParameterString)						// (target_twitter_screen_name)
```

Comment:
I don't treat these assertions as strongly coupled to Decoded, that's why I suggested different name, but I'm not certain about that. Definetly it makes sense to follow the convention so I can name it `a12` as well. Do you think this assertion will stay with us for longer ?

Reply:
I think it will. Even not so long, it won't hurt to follow the convention.

Reply:
`A12` would be the safest name.

As I previously mentioned I'm only a bit concerned about assertions that are quite similar yet still differ a bit. E.g. for the decoded LIT/DOT Holder assertions, they are similar to A4/A7 except that we only count from 2023-01-01 to now. In this sense, giving them a dedicated `A*` name seems overkill. That's why I suggested `Decoded2023_*` series: maybe we can even drop them after the event.

Reply:
If we plan to stay longer with existing assertions we can:

- add another one similiar to A4/A7
- add another one but more parametrized, so it could cover also A4/A7 and mark A4/A7 as deprecated and remove in future.
- change A4/A7 so they can cover new case aswell (potentially breaking change).

If second/third options sounds valuable let's schedule changes.

For now lets stay with new assertion named A*, even for token holding, what you think?

Reply:
Sure let's go for `A*` for now